testing/constraints-3.6.txt

```diff
@@ -19,4 +19,4 @@
 #
 # e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
 # Then this file should have foo==1.14.0
-google-cloud-core==1.3.0
+google-cloud-core==1.4.4
```

Comment:
Delete `testing/constraints-3.6.txt` since the minimum supported runtime is python 3.7

https://github.com/googleapis/python-runtimeconfig/blob/b2297aa3f3f0613da9317c8c01ecdb81ae6c7ecc/setup.py#L87

Reply:
Good catch! Thanks!